### DISCUSSION TITLE
Example of managed integration with shadcn

### DISCUSSION THREAD

**Discussion by the10xProcrastinator** [edmundhung/conform#479] [Q&A]
Was someone able to successfully integrate conform with shadcn components? Looking to adopt conform, only thing stopping me is the direct integration shadcn has with react-hook-forms through their Form component which allows me to not worry about managing the ui for labeling and errors.

**Comment by marilari88** [comment]
@msbeeman  Hi there! I've just completed an example using shadcn-ui. Feel free to check it out in my Pull Request https://github.com/edmundhung/conform/pull/489. Let me know if you have any feedback or questions!

**Reply by musjj** [reply]
Is there a better approach where the component doesn't have to be aware of Conform, so we don't have to make a Conform-specific version of everything?

What's nice about shadcn's `Form` is that it doesn't require a special field component specifically designed for react-hook-form: https://ui.shadcn.com/docs/components/form. You can just drop anything you want into `FormControl` and it'll just work.

Is something like this possible with Conform?
